src/helpers/resizedImages.ts

```diff
@@ -0,0 +1,29 @@
+import ResizedImages from '../models/ResizedImages'
+
+export async function getCachedResize(
+  cid: string,
+  height?: number,
+  width?: number
+): Promise<string | null> {
+  const resizedImage = await ResizedImages.findOne({
+    cid,
+    height: height ? height : null,
+    width: width ? width : null,
```

Comment:
Can the image have only `width || height`? I think it should have both 🤔

Reply:
Yes, it can only have one of the two. (That's the primary use case, we'll likely only provide width)